### ISSUE TITLE
Add time-varying selectivity blocks for WA rec

### ISSUE THREAD

**Issue by kellijohnson-NOAA** [iantaylor-NOAA/Lingcod_2021#61]
- [x] add blocks
- [x] find out if there should be blocks since 2011 (KFJ will email Theresa)

WA Rec CPUE from 2017 had the following blocking:
```
M.dat.81_89$Mgmt[M.dat.81_89$Year >= 1987 ] <- 2
M.dat.90_14$Mgmt[M.dat.90_14$Year >= 1995 ] <- 3
M.dat.90_14$Mgmt[M.dat.90_14$Year >= 1998] <- 4
M.dat.90_14$Mgmt[M.dat.90_14$Year == 1999] <- 5
M.dat.90_14$Mgmt[M.dat.90_14$Year == 2000] <- 6
M.dat.90_14$Mgmt[M.dat.90_14$Year >= 2001] <- 7
M.dat.90_14$Mgmt[M.dat.90_14$Year >= 2007] <- 8
M.dat.90_14$Mgmt[M.dat.90_14$Year >= 2011] <- 9
```

**Comment by kellijohnson-NOAA** [comment]
@melissamonk-NOAA or @aliwhitman I am having trouble trying to use Year + Mgmt where Mgmt is a factor for different time blocks. The parameter estimate for Mgmt is NA and from what I gather it is because of singularity where you cannot have a factor of each Year and then a grouped Year (i.e., Mgmt). Am I doing something wrong here?

**Comment by kellijohnson-NOAA** [comment]
Theresa's response == `22” minimum size requirement was lifted from July 2017 to current for all catch areas but the change happened in the middle of the year`

**Comment by iantaylor-NOAA** [comment]
Initial setup included in commits 39bd40e and 2b9b006. Closing this issue for now.
